internal/blockchain/ethereum/ethereum_test.go

```diff
@@ -47,6 +47,10 @@ func resetConf() {
 	e.InitPrefix(utConfPrefix)
 }
 
+func uuidMatches(id1 *fftypes.UUID) interface{} {
```

Comment:
we've got a safe equals built into UUID you can use (by having a pointer receiver, it works with `nil` pointers on either end - a nice feature of Go)
https://github.com/hyperledger/firefly/blob/25c1039c46820888c2f43eb0bce07feaa4c27fea/pkg/fftypes/uuid.go#L83-L92

Reply:
I copied this helper from some other tests - seems like it can be removed everywhere.